includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -155,11 +155,43 @@ public static function register() {
 			),
 		) );
 
+		add_filter( 'user_has_cap', array( __CLASS__, 'filter_user_has_cap_for_hiding_term_list_table_checkbox' ), 10, 3 );
```

Comment:
Is this filter reset somewhere?

Reply:
No. It should not be reset. This is a permanent filter for the user caps.